### DISCUSSION TITLE
Command gets erased from prompt

### DISCUSSION THREAD

**Discussion by davimmt** [JanDeDobbeleer/oh-my-posh#5212] [Troubleshoot] [answered]
Whenever command A takes a while to execute I use to start early writting command B, when the prompt is not ready yet. On other themes (p10k, or simply zsh/bash), the commands get executed as if they were in line. OhMyPosh just erases them. Can I fix this?

On p10k:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/53799940/1aff4331-9b9d-4308-b505-1a65060e7f75)

On ohmyposh;
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/53799940/81128228-79f7-4c20-8260-f20dd0197d25)

**Comment by JanDeDobbeleer** [comment, marked as answer]
Yes you can. Set `disable_cursor_positioning` to `true` in your configuration.

**Reply by davimmt** [reply]
Thank you! Great job on oh-my-posh, btw. Just started using it when p10k went on life support, thanks to [this guy](https://www.youtube.com/watch?v=9U8LCjuQzdc&t=716s).
